compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/type/builders/BallerinaTupleTypeBuilder.java

```diff
@@ -91,16 +93,20 @@ public TupleTypeSymbol build() {
         return tupleTypeSymbol;
     }
 
-    private BType getMemberType(TypeSymbol memberType) {
+    private BTupleMember getMemberType(TypeSymbol memberType) {
```

Comment:
This is an API change? Let's leave as is for now. cc @dulajdilshan

Reply:
Still not addressed?

Reply:
Should this be kept as `BType`? I thought we should keep the changes in the PR, will change and update.